src/CMakeLists.txt

```diff
@@ -648,6 +665,16 @@ if (PPX_VULKAN)
     endif()
 endif()
 
+#OpenXR libs
+if (PPX_BUILD_XR)
```

Comment:
shouldn't this be in cmake/GraphicsSample ? (along with glfw link infos?)

Reply:
sure, i can move it to there.

Reply:
fixed